pkg/healthcheck/healthcheck.go

```diff
@@ -344,6 +356,33 @@ func (hc *HealthChecker) addLinkerdAPIChecks() {
 		},
 	})
 
+	hc.checkers = append(hc.checkers, &checker{
+		category:    LinkerdControlPlaneExistenceCategory,
+		description: "can successfully call the /Version endpoint",
+		fatal:       true,
```

Comment:
i think this needs `retryDeadline: hc.RetryDeadline`, otherwise `linkerd check` will fail immediately and exit without retry.

you can repro this by running `linkerd install` and then quickly running `linkerd check`...

```bash
$ bin/linkerd check
kubernetes-api: can initialize the client..................................[ok]
kubernetes-api: can query the Kubernetes API...............................[ok]
kubernetes-api: is running the minimum Kubernetes API version..............[ok]
linkerd-existence: control plane namespace exists..........................[ok]
linkerd-existence: controller pod is running...............................[ok]
linkerd-existence: can initialize the client...............................[ok]
linkerd-existence: can successfully call the /Version endpoint.............[FAIL] -- Unexpected API response: 503 Service Unavailable

Status check results are [FAIL]
```